code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2416,6 +2423,9 @@ datum
 				if(d >= blur_start)
 					M.eye_blurry = max(M.eye_blurry, 10/sober_str)
 					M:drowsyness  = max(M:drowsyness, 0)
+				if(d >= vomit_start)
+					if(prob(8))
+						M.fakevomit()
 				if(d >= pass_out)
 					M:paralysis = max(M:paralysis, 20/sober_str)
 					M:drowsyness  = max(M:drowsyness, 30/sober_str)

```

Comment:
Probably 'cause of how old this particular part of the code is---I know Bay wrote this portion.